app/controllers/bulk_updates_controller.rb

```diff
@@ -30,5 +30,17 @@ def new
     @science_keywords = cmr_client.get_controlled_keywords('science_keywords')
   end
 
-  def preview; end
+  def preview
+    redirect_to new_bulk_updates_search_path if request.get?
+
+    add_breadcrumb 'Preview', bulk_update_preview_path
+
+    collections_response = cmr_client.get_collections_by_post({ concept_id: params[:concept_ids], page_size: params[:concept_ids].count }, token)
```

Comment:
I think I've discovered that if `params[:concept_ids]` is empty, this will return **ALL** collections. We have this issue across the application, but in this case, its a real possibility because we don't require collections.